ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/create-run.tsx

```diff
@@ -89,6 +90,32 @@ const formSchema = z.object({
       enabled: z.boolean(),
       formats: z.array(z.string()),
     }),
+    notifier: z.object({
+      enabled: z.boolean(),
+      notifierRules: z.string().optional(),
+      resolutionsFile: z.string().optional(),
+      mail: z.object({
+        recipientAddresses: z.array(z.object({ email: z.string() })),
+        mailServerConfiguration: z.object({
+          hostName: z.string(),
+          // This primitive type integer) should be coerced to avoid the error
+          // "Expected number but received string" when the form is submitted.
+          // See: https://github.com/react-hook-form/resolvers/issues/73#issuecomment-1647321322
+          port: z.coerce.number().int(),
```

Comment:
`coerce` is quite basic Zod functionality, so this code comment could maybe be removed.